src/javascript/app/common/redirect_banner.js

```diff
@@ -12,14 +13,30 @@ const RedirectBanner = (() => {
             const eu_country = isEuCountrySelected(Client.get('residence')) || isEuCountrySelected(State.getResponse('website_status.clients_country'));
             if (eu_country) {
                 handleRedirect();
+            } else {
+                handleRedirectROW();
             }
             
         });
 
     };
 
     const handleRedirect = () => {
-        window.location.href = '/move-to-deriv/';
+        window.location.href = 'https://www.binary.bot/movetoderiv.html';
+    };
+    const handleRedirectROW = () => {
+
+        // Check if param have ?binary-com-lp
+        const redirectBinary = new URLSearchParams(window.location.search);
+
+        if (redirectBinary.has('binary-com-lp') || Cookies.get('binary-com-show')){
```

Comment:
I think we don’t need this `binary-com-lp` check here anymore cause the changes will be in binary itself now.
We just need to append `/binary-to-deriv/` once the client country is rest of the world.

@habib-deriv please correct me if I'm wrong 🙏